app-arch/innoextract/innoextract-1.8.recipe

```diff
@@ -55,6 +55,8 @@ BUILD()
 {
 	export CXXFLAGS="-D_BSD_SOURCE"
 	cmake . $cmakeDirArgs \
+		-DBoost_NO_BOOST_CMAKE=ON \
+		-DCMAKE_BUILD_TYPE=Release \
 		-DCMAKE_INSTALL_BINDIR=$commandBinDir
 	make $jobArgs
 }
```

Comment:
Nice catch! Suggestion:
```
	export CXXFLAGS="-D_BSD_SOURCE"
	cmake -Bbuild -S. $cmakeDirArgs \
		-DCMAKE_BUILD_TYPE=Release \
		-DCMAKE_INSTALL_BINDIR=$commandBinDir \
		-DBoost_NO_BOOST_CMAKE=ON \
	make -C build $jobArgs
```
First system variables, then user variables (I know there is no straight guideline for this (yet). :)
In INSTALL() you could use `make -C build install` then.

Reply:
Thanks! I've made the suggested changes, and also updated to version 1.9. It adds support for building on newer Boost versions which could be useful in the future.